### ISSUE TITLE
ofxUIOFWrapper includes nightly build methods

### ISSUE THREAD

**Issue by kimasendorf** [rezaali/ofxUI#251]
Hey Reza, thx for the fix.

Now it looks like ofxUIOFWrapper makes use of ofDrawCircle etc. So we lost compatibility to 0.8.4

=)

**Comment by kimasendorf** [comment]
ofxUICanvas seems also to include the new syntax for loading fonts.

**Comment by rezaali** [comment]
@kimasendorf yeah, it did, I'm happy to make it 0.8.4 compatible, but that means more code and more upkeep, pull requests are welcome.
